Beginner Instructions.md

```diff
@@ -0,0 +1,104 @@
+## These instructions are for a beginner to set up a ML201136 board on a 3 phase supply, using the components below
+
+From this forum comes the source of the setup information: https://community.home-assistant.io/t/another-power-monitoring-solution-uk-mottramlabs-ct-clamp-board/338139
+
+
+## Components
+ - 1x ML201136 (Blue Board)
+- 4x SCT013 clamp meters
+- 1x ESP8266 (Wemos D1 Pro)
+- 1x 12V 2A Power supply
+
+
+## Assembly and configuration
+
+Bear in mind that this is the first time I've used ESPHome so your experience may exceed mine, also I may have a few glaring holes in my setup, but that will hopefully be fixed upon creating more or receiving advice.
+
+1. Plug the Wemos into the board, and power it via the 12V cable.
+2. Plug a USB cable into the Wemos, and your computer.
+3. Install ESPHome using the documentation here: https://www.home-assistant.io/integrations/esphome/
+4. Adding a device to ESPHome can be done a few ways, I'd recommend getting familiar with the official documentation before continuing here: https://esphome.io/guides/getting_started_hassio.html
+5. At the end of this, you should have a board name, and it should be on your wireless network with a secure setup.
+6. You may have another way of doing this, but I use my Windows PC to browse to the SMB share, if you've not set this up, you can add it via Add-ins: https://github.com/home-assistant/addons/tree/master/samba
+7. Use Windows Explorer to open Home Assistant from Windows, find 'config', then 'esphome'. 
+8. A .yaml file will be there with the name you just gave it.
+9. Open it, and paste the text from the configuration file below what's already there.
```

Comment:
Hi! Could you also provide the "configuration file below"? I can't seem to find it.

Reply:
Never mind, found it here: https://github.com/Mottramlabs/4-Channel-Mains-Current-Sensor-ESP8266/pull/7